lib/guard/redis.rb

```diff
@@ -92,6 +95,7 @@ def config
     end
 
     def pid
+      return @pid unless @pid
```

Comment:
You're entirely correct. Couldn't make my mind up between `unless @pid.nil?`and that. Sorted now 👍